lib/web3.js

```diff
@@ -99,7 +104,8 @@ var setupMethods = function (obj, methods) {
 /// setups api calls for these properties
 var setupProperties = function (obj, properties) {
     properties.forEach(function (property) {
-        var proto = {};
+        var objectProperties = property.name.split('.'),
+            proto = {};
         proto.get = function () {
```

Comment:
would be better to do:

```javascript
var name = property.name;
var objectProperties = name.split('.');
if (objectProperties.length > 1) {
  obj = objectProperties[0];
  name = objectProperties[1];
}
```

also please be consistent with defining variables in the rest of the code.
```javascript
var objectProperties = property.name.split('.');
var proto = {};
```

would be better